libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/SalesforceListActivity.java

```diff
@@ -50,6 +51,7 @@
 
 	private PasscodeManager passcodeManager;
     private UserSwitchReceiver userSwitchReceiver;
+    private LogoutCompleteReceiver logoutCompleteReceiver;
```

Comment:
Should we do the same in SalesforceReactActivity (it's a subclass of ReactActivity)